client/components/backup-storage-space/usage-warning/style.scss

```
@@ -60,3 +58,24 @@
 		font-weight: 600;
 	}
 }
+
+.usage-warning__upsell-price {
+	&.is-placeholder {
+		@include placeholder( --color-neutral-10 );
+	}
+	.display-price {
+		display: inline;
+	}
+	.plan-price {
+		font-size: 1rem;
+	}
+	.plan-price__integer,
+	.plan-price__fraction,
+	.plan-price__currency-symbol {
+		font-size: 1rem;
+	}
+	.display-price__billing-time-frame {
+		font-size: 1rem;
+		color: var(--color-text);
+	}
+}
```

This is absolutely and completely inconsequential, but conventionally a block like this would be formatted like so in Calypso:

```suggestion
.usage-warning__upsell-price {
	&.is-placeholder {
		@include placeholder( --color-neutral-10 );
	}
	
	.display-price {
		display: inline;
	}
	
	.plan-price {
		font-size: 1rem;
	}
	
	.plan-price__integer,
	.plan-price__fraction,
	.plan-price__currency-symbol {
		font-size: 1rem;
	}
	
	.display-price__billing-time-frame {
		font-size: 1rem;
		color: var( --color-text );
	}
}
```

---

Ah! Thanks.  I am not sure why prettier didn't format the styles automatically.